src/hct_mis_api/apps/household/serializers.py

```diff
@@ -18,10 +17,10 @@ def get_household_status(household: Household) -> Tuple[str, datetime]:
         if payments.exists():
             return "paid", payments.first().delivery_date
 
-    selections = HouseholdSelection.objects.filter(household=household)
+    selections = Payment.objects.filter(household=household)
```

Comment:
These statuses doesn't make sense now. 
Payment.objects.filter(household=household) - this filtering is duplicated.

if selection.parent.status == PaymentPlan.Status.TP_PROCESSING:
            return "sent to cash assist", selection.updated_at # "sent to cash assist"???

Reply:
updated  just to show if HH was paid and if parent in pre-PaymentPlan states then show when was targeted